packages/dashboard-hardhat-plugin/README.md

```diff
@@ -0,0 +1,58 @@
+# @truffle/dashboard-hardhat-plugin
+
+Use [Truffle Dashboard](https://trufflesuite.com/docs/truffle/how-to/use-the-truffle-dashboard/) with your Hardhat projects.
+
+## What
+
+This plugin brings [Truffle Dashboard](https://trufflesuite.com/docs/truffle/how-to/use-the-truffle-dashboard/) to your Hardhat projects. It extends `npx hardhat compile` by sending the compiled artifacts to Truffle Dashboard, enabling you to sign transactions with your browser-based wallet. No more floating private keys!
+
+## Installation
+
+You can install this plugin with `npm` (or `yarn`) by running:
+
+```bash
+npm i @truffle/dashboard-hardhat-plugin -D
+```
+
+Beyond that, simply import the plugin in your `hardhat.config.js`:
+
+```ts
+import "@truffle/dashboard-hardhat-plugin";
+```
+
+## Setup
+
+This extension assumes you have `truffle` installed (either globally or in a local project context). If not, you can install it with `npm i -g truffle`. Beyond this you'll be able to start Truffle Dashboard with `truffle dashboard`.
```

Comment:
Maybe somewhere in here we should link to the [Use Truffle Dashboard](https://trufflesuite.com/docs/truffle/how-to/use-the-truffle-dashboard/) docs.